### DISCUSSION TITLE
Formula issues with column names with a space in their names

### DISCUSSION THREAD

**Discussion by ovizii** [nocodb/nocodb#4591] [Q&A]
I found this thread mentioning a similar problem but no solution for me:  https://github.com/nocodb/nocodb/discussions/2805

Here are my system infos: 

Node: **v16.17.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.100.1**

I just created a new formula column, lets let pictures speak: 

![image](https://user-images.githubusercontent.com/837902/205390497-4275e6f7-5488-408c-b4d9-2f9dc2cfa454.png)

After I click on the column named "Spalte2" all is well. 

![image](https://user-images.githubusercontent.com/837902/205390777-e329edb6-8cba-433d-bffb-bc02e72e84a0.png)

If I however select the column named "Spalte 3" with a SPACE in the name: 

![image](https://user-images.githubusercontent.com/837902/205391207-f4764a98-cbaf-47fe-8af7-3c2adddf58e6.png)

Mind you, I tried typing {Spalte 3} as well as just clicking it in the drop-down with the same result.

**Comment by dstala** [comment]
I tried on `safari` browser : Version 14.1.2 (16611.3.10.1.6). Worked fine.

**Comment by ovizii** [comment]
Thanks for the pointer, I had the issue with: Version 107.0.5304.123 (Official Build) (64-bit)

Will try now with: Firefox 107.0.1 

Firefox has the same problem as reported above.

**Reply by wingkwong** [reply]
I can reproduce the issue on Chrome. Let's track the issue [here](https://github.com/nocodb/nocodb/issues/4592).
